index.html

```diff
@@ -0,0 +1,102 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link href="index.css" rel="stylesheet" />
+    <title>Document</title>
+  </head>
+
+  <body>
+    <div id="main">
+      <h1 id="title">Dr. Norman Borlaug</h1>
+      <div id="img-div">
+        <img alt="img" id="image" src="img/10613180113_fdf7bcd316_b.jpg" />
+        <p id="img-caption">
+          Dr. Norman Borlaug, third from the left, trains biologists in Mexico on how to increase
+          wheat yields - part of his life-long war on hunger.
+        </p>
+      </div>
+      <div>
```

Comment:
Use section instead of div

Reply:
- It would be more polite and descriptive to write:
- [ ] - Please use `section` tag instead of `div` tag on this element. This is to make your markup more semantic and support SEO.
- This kind of message will be much clearer to the student/developer on why they should use semantic tags.